receiver/mongodbatlasreceiver/README.md

```diff
@@ -23,6 +23,10 @@ receivers:
   mongodbatlas:
     public_key: ${MONGODB_ATLAS_PUBLIC_KEY}
     private_key: ${MONGODB_ATLAS_PRIVATE_KEY}
-```
-
 
+service:
+  pipelines:
+    metrics:
+      receivers: [mongodbatlas]
```

Comment:
you need an exporter as well for this... so probably better to not have this at all.

Reply:
Related to #6884